packages/node_modules/@node-red/nodes/core/network/31-tcpin.js

```diff
@@ -371,7 +371,6 @@ module.exports = function(RED) {
             var server = net.createServer(function (socket) {
                 socket.setKeepAlive(true,120000);
                 if (socketTimeout !== null) { socket.setTimeout(socketTimeout); }
-                var remoteDetails = socket.remoteAddress+":"+socket.remotePort;
```

Comment:
Why is this changed ? Nothing to do with csv ?

Reply:
Sorry. As you say I noticed now that this action should separate. I forgot to make new branch about it. I intend to make new branch.